proto/streamlit/proto/TextArea.proto

```diff
@@ -19,5 +19,5 @@ syntax = "proto3";
 message TextArea {
   string id = 1;
   string label = 2;
-  string default = 3;
+  string value = 3;
```

Comment:
Like Button.proto comment: To change the protobuf, choose a new numerical value for "value" and explicitly deprecate the word "default" as well as its number.